File: JadenCase.js
String.prototype.toJadenCase = function () {
  
  var individ_words = this.split(" ");
  var final = "";
  
  for (word of individ_words) {
    word = word.charAt(0).toUpperCase() + word.slice(1);
    final += word + " ";
  }
  
  return final.slice(0,-1)
  
};

File: JadenCase.java
public class JadenCase {

	public String toJadenCase(String phrase) {
    if (phrase.isEmpty() || phrase == null) { return null; }
    
		String[] individ_words = phrase.split(" ");
    String result = "";
    
    for (String word : individ_words) {
      result += Character.toUpperCase(word.charAt(0)) + word.substring(1) + " ";
    }
		return result.substring(0, result.length()-1);
	}

}